src/query/shorthand.ts

```diff
@@ -0,0 +1,88 @@
+import {Type} from 'vega-lite/src/type';
+
+import {EncodingQuery} from './encoding';
+import {SpecQuery, stack} from './spec';
+import {enumSpecShort, isEnumSpec, SHORT_ENUM_SPEC} from './query';
+
+import {getNestedEncodingPropertyChild, Property} from '../property';
+import {keys} from '../util';
+
+export function spec(specQ: SpecQuery): string { // stringifySpecQuery
```

Comment:
Don't include comment like

```
// stringifySpecQuery
```

because in the future no one besides you and me would know that you're referring to the old method that you're removing so this is useless comment.  

Make sure to remove similar comments throughout this PR.